### PR TITLE
JBIDE-24273 - CDK3 server adapter editor: can save editor even if Min…

### PR DESCRIPTION
**Pull request by robstryker**
…ishift Location is invalid

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/MinishiftLocationSection.java [hunk 1]
```diff
@@ -248,4 +249,15 @@ private String getErrorString() {
 		}
 		return null;
 	}
+	
+	public IStatus[] getSaveStatus() {
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/MinishiftLocationSection.java]: @robstryker dare to address this issue, please?

**Comment by adietish** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by rhopp** [comment]
"Do we have bot-tests for this? if not, we should jira that" @odockal?

**Comment by odockal** [comment]
@adietish @rhdevelopers-ci @robstryker I count with that. See https://issues.jboss.org/browse/JBIDE-24660.

**Comment by odockal** [comment]
@robstryker @adietish PR covering this functionality sent. https://github.com/jbosstools/jbosstools-integration-tests/pull/1840. And I also tested this manually. Seems alright. +1 from m side!

**Pull request merged**